### ISSUE TITLE
Web page is not being navigated sequentially with the Tab key, Publish issue.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#820]
**Describe the bug**
Web page is not being navigated sequentially with the Tab key, Publish issue. The sequence of focus is unordered.

**To Reproduce**

Go to Publish  (http://localhost:9992/cm/app/?view=publish&pathType=unknown)
Use Tab key to focus on different components of the screen.
Focus is not moving in sequence on Card, List, Select a site etc.

**Expected behavior**
Web page should be navigated sequentially with the Tab key.

**Additional context**
Navigate all the options available for similar issue and fix.

**Comment by ramdev1980** [comment]
Fixed with https://github.com/percussion/percussioncms/issues/780
